textx/metamodel.py

```diff
@@ -515,36 +564,44 @@ def register_obj_processors(self, obj_processors):
         self.obj_processors = obj_processors
 
 
-def metamodel_from_str(lang_desc, metamodel=None, **kwargs):
+def metamodel_from_str(lang_desc, metamodel=None, enable_global_model_repository=False, **kwargs):
```

Comment:
I think that `enable_global_model_repository` should be treated like the rest of metamodel parameters, thus be a part of `**kwargs` and handled in constructor of `TextXMetaModel` class. In that case, `metamodel_from_str` and `metamodel_from_file` would not have to be changed at all.

Reply:
Futhermore I'm always for shorter names if possible: `enable_global_model_repository` -> `global_repository`

Reply:
Parameter name changed as suggested. Moved logic in TextXMetaModel constructor as suggested. Adapted scoping.md according to changes.